src/Gearer.php

```diff
@@ -8,22 +8,21 @@
 
 class Gearer
 {
-    const API_URL = 'https://gateway.gear.mycelium.com';
-
-
-    public $gatewayId;
-    public $gatewaySecret;
-    public $client;
+    private $baseUrl;
+    private $gatewayId;
+    private $gatewaySecret;
+    private $client;
+    private $endpoint;
```

Comment:
we should try to avoid exposing variables when possible/needed, in case of need should use getters/setter to have more control